flaskapi/demo/Dockerfile

```diff
@@ -0,0 +1,20 @@
+# context should be set to dir above demo dir
+FROM python:3.9-alpine
+
+# security fix for expat
+RUN apk upgrade expat
+
+RUN apk add build-base curl libpq-dev
```

Comment:
Can some or all of these packages be removed after the build is complete? Though it doesn't matter that much for a demo I suppose. But might be good to do it anyway to provide a reference Dockerfile for package users.

Reply:
We do not need curl anymore anymore.  We need libpq, but can safely delete build-base after the build

Reply:
You probably only need libpq rather than libpq-dev I'm guessing.

Reply:
This is now outdated as demo is no longer included here